subscription-service/build.gradle.kts

```diff
@@ -25,6 +25,7 @@ dependencies {
 
     developmentOnly("org.springframework.boot:spring-boot-devtools")
 
+    runtimeOnly("org.flywaydb:flyway-database-postgresql")
```

Comment:
There was no flyway before on subscription service?

Reply:
In Flyway 10 (which comes with Spring Boot 3.3), they moved the DB drivers out of the core distribution. So they now have to be explicitely declared.